controller/search_blackbox_test.go

```diff
@@ -1054,7 +1052,6 @@ func (s *searchControllerTestSuite) TestSearchQueryScenarioDriven() {
 		)
 		_, result := test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
 		require.NotEmpty(t, result.Data)
-		assert.Len(t, result.Data, 1)
```

Comment:
Why was this removed?

Reply:
@jarifibrahim because the test query was too broad and this expectation is not always true.